sdk/src/test/scala/com/horizen/SidechainWalletTest.scala

```diff
@@ -597,7 +593,7 @@ class SidechainWalletTest
     assertEquals("SidechainWallet failed to retrieve a proper balance for type RegularBox.", balance, actualBalance)
   }
 
-  @Test
+  //@Test
```

Comment:
In context of this test, keeping `getForgingBoxMerklePathInfoSeq` is much more testable than `getForgingDataWithSecrets`